lib/dom_global_attributes.ml

```diff
@@ -0,0 +1,29 @@
+include Dom_dsl_core.Prop
+
+let accessKey = string "accessKey"
+
+let className = string "className" (* substitute for "class" *)
+
+let contentEditable = bool "contentEditable"
+
+let contextMenu = string "contextMenu"
+
+let dir = string "dir" (* "ltr", "rtl" or "auto" *)
+
+let draggable = bool "draggable"
+
+let hidden = bool "hidden"
+
+let id = string "id"
+
+let lang = string "lang"
+
+let role = string "role" (* ARIA role *)
+
+let style = (any "style" : Dom.Style.t -> t)
+
+let spellCheck = bool "spellCheck"
+
+let tabIndex = int "tabIndex"
```

Comment:
What is the rule about camelCase / snake_case in general? I assume for DOM related stuff we keep the original names? cc @glennsl

Reply:
Good question. HTML attributes are actually flatcase, so if we were restricted to that we could just ignore the issue altogether. Unfortunately our life is not that simple. SVG actually manages to mix both flatcase, cemaelCase _and_ kebab-case in its attribute naming. And React of course has chosen camelCase even for flatcased names. Since there are other React-specific naming conventions we rely on as well (`className`, `htmlFor` etc.), it might be better to just stick with that, yeah.

Now I wonder if we should have stuck with camelCase for CSS properties too...